diff --git a/internal/handler/group_handler.go b/internal/handler/group_handler.go
index aa4cef1..dea49d4 100644
--- a/internal/handler/group_handler.go
+++ b/internal/handler/group_handler.go
@@ -84,8 +84,8 @@ func isValidGroupName(name string) bool {
 	if name == "" {
 		return false
 	}
-	// 允许使用小写字母、数字、下划线和中划线，长度在 3 到 30 个字符之间
-	match, _ := regexp.MatchString("^[a-z0-9_-]{3,30}$", name)
+	// 允许使用小写字母、数字、下划线和中划线，长度在 3 到 100 个字符之间
+	match, _ := regexp.MatchString("^[a-z0-9_-]{3,100}$", name)
 	return match
 }
 
@@ -182,7 +182,7 @@ func (s *Server) CreateGroup(c *gin.Context) {
 	// Data Cleaning and Validation
 	name := strings.TrimSpace(req.Name)
 	if !isValidGroupName(name) {
-		response.Error(c, app_errors.NewAPIError(app_errors.ErrValidation, "无效的分组名称。只能包含小写字母、数字、中划线或下划线，长度3-30位"))
+		response.Error(c, app_errors.NewAPIError(app_errors.ErrValidation, "无效的分组名称。只能包含小写字母、数字、中划线或下划线，长度3-100位"))
 		return
 	}
 
@@ -350,7 +350,7 @@ func (s *Server) UpdateGroup(c *gin.Context) {
 	if req.Name != nil {
 		cleanedName := strings.TrimSpace(*req.Name)
 		if !isValidGroupName(cleanedName) {
-			response.Error(c, app_errors.NewAPIError(app_errors.ErrValidation, "无效的分组名称格式。只能包含小写字母、数字、中划线或下划线，长度3-30位"))
+			response.Error(c, app_errors.NewAPIError(app_errors.ErrValidation, "无效的分组名称格式。只能包含小写字母、数字、中划线或下划线，长度3-100位"))
 			return
 		}
 		group.Name = cleanedName
